src/profile-logic/gecko-profile-versioning.js

```diff
@@ -59,6 +59,7 @@ function _archFromAbi(abi) {
 
 // _upgraders[i] converts from version i - 1 to version i.
 // Every "upgrader" takes the profile as its single argument and mutates it.
+/* eslint-disable no-useless-computed-key */
```

Comment:
unfortunately flow doesn't like number properties; but maybe we should use simple strings here, because that's what they are converted to, right ?